File: LogicBuilding_Java/Program4.java
//program to print addition of two numbers using function
//Dynamic values logic inside the seperate function in same class

import java.util.*;

class Program4
{
    public static int Addition(int iValue1,int iValue2)
    {
        int iSum = 0;
        iSum = iValue1 + iValue2;
        return iSum;
    }

    public static void main(String arg[])
    {
        int iNo1 = 0, iNo2 = 0, iAns = 0;

        Scanner sc = new Scanner(System.in);

        System.out.println("Enter first number : ");
        iNo1 = sc.nextInt();

        System.out.println("Enter second number : ");
        iNo2 = sc.nextInt();

        iAns = Addition(iNo1, iNo2);

        System.out.println("Addition is : "+iAns);

    }
}

File: LogicBuilding_C/program4.c
//program to print Addition of two numbers take input from user

#include<stdio.h>
int main()
{
    int iNo1 = 0;
    int iNo2 = 0;
    int iAns = 0;

    printf("Enter first number : ");
    scanf("%d",&iNo1);

    printf("Enter second number : ");
    scanf("%d",&iNo2);

    iAns = iNo1 + iNo2;
    printf("Addition of two numbers is : %d",iAns);

    return 0;
}

/*
    %d = int
    %f = float
    %c = character
*/
